content/notebooks/data_discovery_and_access/data_discovery_and_access.ipynb

```diff
@@ -232,7 +237,9 @@
    "source": [
     "## Streaming from the Roman Science Platform S3 Bucket\n",
     "\n",
-    "Though Roman data will eventually be available through MAST, we currently offer a small set of simulated data available in a separate S3 bucket. These files can be streamed in exactly the same way as the JWST FITS file above. Additionally, we can browse the available files similarly to a Unix terminal. A full list of commands can be found in the `s3fs` documentation [here](https://s3fs.readthedocs.io/en/latest/api.html#)."
+    "Though Roman data will eventually be available through MAST, we currently offer a small set of simulated data available in a separate S3 bucket. These files can be streamed in exactly the same way as the JWST FITS file above. Additionally, we can browse the available files similarly to a Unix terminal. A full list of commands can be found in the `s3fs` documentation [here](https://s3fs.readthedocs.io/en/latest/api.html#).\n",
+    "\n",
+    "Because the S3 bucket is specific to the science platform, it is not publicly available. We have managed the permissions so none need to be specified explicitly, however we do need to create a new `S3FileSystem` with the new permissions. Because of the permissions, the cell below will not work on a private comuter without modification to include the correct permissions."
```

Comment:
It's more correct to say that the S3 bucket is currently not open to the public outside of the science platform. I would not say that you can modify the cell to work on a private computer with modifications. I don't actually believe that to be the case, since my understanding is that the S3 bucket is just not visible outside the platform.